### DISCUSSION TITLE
arm64/V8

### DISCUSSION THREAD

**Discussion by chaomarks** [louislam/dockge#496] [General]
The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Can you make a (linux/arm64/v8) version, please

**Comment by louislam** [comment]
Please check again, it should be already available. 

https://hub.docker.com/layers/louislam/dockge/1/images/sha256-079ee64bbc83a44d9cc9044335b0a9ecc4cbd17329df61c556ca567b9c7ae36a?context=explore

**Comment by suranyami** [comment]
I tried it on `arm64` (DietPi, Raspberry Pi 4) and it worked without any issues.

@chaomarks How did you set it up?
